### PR TITLE
vnetPeering Module Hygiene - Backlog Item 106634

### PR DESCRIPTION
**Pull request by JamJarchitect**
# Overview/Summary

Backlog Item 106634 - virtualNetworkPeer module changed to vnetPeering to conform to code hygiene standards

## This PR fixes/adds/changes/removes

1. Replaces the 'virtualNetworkPeering' module to 'vnetPeering'
2. Removed comment block synopsis from module in prep for Backlog Item 83064

### Breaking Changes

1. change of module name from virtualNetworkPeering to vnetPeering

## Testing Evidence

image

## As part of this Pull Request I have

- [X] Read the [Contribution Guide](https://github.com/Azure/ALZ-Bicep/wiki/Contributing) and ensured this PR is compliant with the guide
- [X] Checked for duplicate [Pull Requests](https://github.com/Azure/ALZ-Bicep/pulls)
- [N/A] Associated it with relevant [GitHub Issues](https://github.com/Azure/ALZ-Bicep/issues)
- [ADO 106634] *(ALZ Bicep Core Team Only)* Associated it with relevant [ADO Items](https://aka.ms/alz/bicep/backlog)
- [X] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/ALZ-Bicep/tree/main)
- [X] Performed testing and provided evidence.
- [N/A] Updated tests *(if required)* [Unit](https://github.com/Azure/ALZ-Bicep/blob/main/.github/workflows/bicep-build-to-validate.yml) - [Linting](https://github.com/Azure/ALZ-Bicep/tree/main/.github/workflows) - [E2E (End-To-End)](https://github.com/Azure/ALZ-Bicep/blob/main/tests/pipelines/bicep-build-to-validate.yml)
- [X] Updated relevant and associated documentation (e.g. Contribution Guide, Module READMEs, Wiki Docs etc.)
- [N/A] If relevant, created or updated Code Tours [here](https://github.com/Azure/ALZ-Bicep/blob/main/.vscode/tours)

### CODE REVIEW

File: infra-as-code/bicep/modules/vnetPeering/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Module: Virtual Network Peer
+# Module: VNet Peering
 
 This module creates a virtual network peer between two virtual networks and is to be utilized by other modules.  Module will need to be called twice to create the completed peering.  Each time with a peering direction. This allows for peering between different subscriptions.
```
Comment (jtracey93) [inline infra-as-code/bicep/modules/vnetPeering/README.md:3]: Can we update the first sentence to talk about virtual network peering instead of peer?

**Review by jtracey93 — CHANGES_REQUESTED** [review]
Nice work here @JamJarchitect just one small thing to change

File: infra-as-code/bicep/modules/vnetPeering/README.md (hunk 1 above)
Comment (JamJarchitect) [reply infra-as-code/bicep/modules/vnetPeering/README.md:3]: grammatical change has been made on latest commit [39e476a]

**Review by jtracey93 — APPROVED** [review]

**Pull request merged**